wagtail/admin/wagtail_hooks.py

```diff
@@ -127,6 +127,59 @@ def register_workflow_tasks_menu_item():
     return WorkflowTasksMenuItem(_('Workflow tasks'), reverse('wagtailadmin_workflows:task_index'), icon_name='thumbtack', order=150)
 
 
+@hooks.register('register_bulk_action_filters')
+def bulk_action_filters():
+    yield Button(
+        _('All'),
+        '?filters=',
+        attrs={'title': _("All pages"), 'filter': _("")},
+        priority=10
+    )
+
+    yield Button(
+        _('Status: Draft'),
+        '?filters=status:draft',
+        attrs={'title': _("Draft pages"), 'filter': _("status:draft")},
+        priority=20
+    )
+
+    yield Button(
+        _('Status: Live'),
+        '?filters=status:live',
+        attrs={'title': _("Live pages"), 'filter': _("status:live")},
+        priority=30
+    )
+
+
+@hooks.register('register_bulk_action_choices')
+def bulk_action_choices(page, is_parent=False, next_url=None):
+    if page:
+        yield PageListingButton(
+            _('Move'),
+            reverse('wagtailadmin_bulk_delete', args=[page.id]) + '?' + urlencode({'next': next_url}),
```

Comment:
Looks like this should be `wagtailadmin_bulk_move` which has not been implemented yet.